OpenVpnService.csproj

```diff
@@ -10,7 +10,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>OpenVpnService</RootNamespace>
     <AssemblyName>OpenVpnServ2</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>

```

Comment:
This causes a warning in the Ubuntu build:

    warning : TargetFrameworkVersion 'v4.8' not supported by this toolset (ToolsVersion: 14.0).

According to https://www.mono-project.com/docs/about-mono/compatibility/ mono supports only v4.7. Should we use that instead or do we need v4.8?

Reply:
okay